example/basic_example/main.go

```diff
@@ -18,12 +19,13 @@ func main() {
 	dataset := flag.String("dataset", "opentelemetry", "Your Honeycomb dataset")
 	flag.Parse()
 
-	exporter := honeycomb.NewExporter(honeycomb.Config{
+	exporter, err := honeycomb.NewExporter(honeycomb.Config{
 		ApiKey:      *apikey,
 		Dataset:     *dataset,
 		Debug:       true,
 		ServiceName: "opentelemetry-basic-example",
 	})
+	log.Fatal(err)
```

Comment:
You sure you want this unconditional rather than `if != nil`?

Reply:
whoops! Yeah. Probably don't want to 💥 every time.